packages/petr4/petr4.0.1/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "Petr4: Formal Semantics for the P4 Programming Language"
+maintainer: "jnfoster@cs.cornell.edu"
+authors: ["Nate Foster <jnfoster@cs.cornell.edu>"]
+homepage: "https://github.com/cornell-netlab/petr4"
+bug-reports: "https://github.com/cornell-netlab/petr4/issues"
+dev-repo: "git+https://github.com/cornell-netlab/petr4/"
+license: "Apache2"
+build: [
+  ["dune" "build" "--root" "." "-j" jobs "@install"]
+]
+depends: [
+  "ocaml" { build & >= "4.05.0" }
```

Comment:
```suggestion
  "ocaml" { >= "4.05.0" }
```
OCaml is not a build dependency in most cases.

Reply:
Thanks. Our mistake.